### PR TITLE
Update Documentation for 'Volumio' Media Player Component Configuration

### PR DESCRIPTION
**Pull request by vaidyasm**
**Description:**
Using new config vars
Related to Issue #6385

Pull request in [home-assistant](https://github.com/home-assistant/home-assistant): https://github.com/home-assistant/home-assistant.io/pull/7030

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/media_player.volumio.markdown [hunk 1]
```diff
@@ -26,8 +26,17 @@ media_player:
...
-- **name** (*Optional*): Name of the device
-- **host** (*Required*): IP address or hostname of the device
-- **port** (*Required*): Port number of Volumio service
+{% configuration %}
+name:
+  description: Name of the device
+  required: false
+  type: string
+host:
+  description: IP address or hostname of the device
```
Comment (tjorim) [inline source/_components/media_player.volumio.markdown]: The IP [...].

File: source/_components/media_player.volumio.markdown [hunk 2]
```diff
@@ -26,8 +26,17 @@ media_player:
...
+  required: true
+  type: string
+port:
+  description: Port number of Volumio service
```
Comment (tjorim) [inline source/_components/media_player.volumio.markdown]: The .

File: source/_components/media_player.volumio.markdown [hunk 3]
```diff
@@ -26,8 +26,17 @@ media_player:
...
 ```
 
-Configuration variables:
-
-- **name** (*Optional*): Name of the device
-- **host** (*Required*): IP address or hostname of the device
-- **port** (*Required*): Port number of Volumio service
+{% configuration %}
+name:
+  description: Name of the device
```
Comment (tjorim) [inline source/_components/media_player.volumio.markdown]: The name of the device.

File: source/_components/media_player.volumio.markdown [hunk 4]
```diff
@@ -26,8 +26,17 @@ media_player:
...
+  required: false
+  type: string
+host:
```
Comment (tjorim) [inline source/_components/media_player.volumio.markdown:35]: Try to cross-check the documentation with the actual code.
In this case you can find the default values here: https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/media_player/volumio.py#L49
`default: Volumio`

File: source/_components/media_player.volumio.markdown [hunk 5]
```diff
@@ -26,8 +26,17 @@ media_player:
...
+  description: IP address or hostname of the device
+  required: true
+  type: string
+port:
```
Comment (tjorim) [inline source/_components/media_player.volumio.markdown:40]: Default value for host is localhost (https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/media_player/volumio.py#L32)

File: source/_components/media_player.volumio.markdown [hunk 6]
```diff
@@ -26,8 +26,17 @@ media_player:
...
+  description: Port number of Volumio service
+  required: true
+  type: integer
+{% endconfiguration %}
```
Comment (tjorim) [inline source/_components/media_player.volumio.markdown:45]: Same for port

**Review by tjorim — CHANGES_REQUESTED** [review]
Thanks for helping us out with this PR! I left some feedback.

**Comment by vaidyasm** [comment]
@tjorim Thanks for your feedbacks. They have been incorporated. It helps to know the source where the defaults were defined.

**Comment by frenck** [comment]
Rebased this PR onto `current`.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thank you for this PR, @vaidyasm! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**